src/MenuItems/MenuItems.js

```diff
@@ -0,0 +1,30 @@
+import MenuItem from "../MenuItem/MenuItem";
+
+import { useState } from "react";
+
+const MenuItems = ({menuItems}) => {
+    const [searchInput, setSearchInput] = useState('');
+
+    let dataToDisplay = menuItems;
+
+    if(searchInput){
+        dataToDisplay = menuItems.filter(item => {
+            let {name, shortDescription} = item;
+            return name.toLowerCase().includes(searchInput.toLowerCase()) || shortDescription.toLowerCase().includes(searchInput.toLowerCase())
```

Comment:
This is a small thing that won't affect real life performance, but notice that this line of code calls `searchInput.toLowerCase()` twice in a row (on each side of the or statement). Strictly speaking, this isn't efficient. Imagine if the `searchInput` string was a trillion characters long. It would take a very long time to lowercase that string, and we'd have to do it twice. Alternatively, you could call `searchInput.toLowerCase()` once and assign it to a variable before line 13, then reuse that variable in line 13.